lib/active_merchant/billing/gateways/openpay.rb

```diff
@@ -130,10 +130,22 @@ def add_creditcard(post, creditcard, options)
             holder_name: creditcard.name
           }
           add_address(card, options)
+          add_customer_data(post, creditcard, options)
           post[:card] = card
         end
       end
 
+      def add_customer_data(post, creditcard, options)
+        if options[:email]
+          customer = {
+            name: options[:name] || creditcard.name,
```

Comment:
I know it's what was being used before, but pulling name straight from options is a pretty rare pattern. Since this method is called only when adding a credit card, I think the preference order here should be switched.  Otherwise, LGTM 👍